File: integration-tests-java/src/main/java/io/micronaut/jimmer/java/it/service/IUserRoleService.java
package io.micronaut.jimmer.java.it.service;

import io.micronaut.jimmer.java.it.entity.UserRole;
import java.util.UUID;

public interface IUserRoleService {

    UserRole findById(UUID id);

    void updateById(UUID id);

    void deleteById(UUID id);

    UserRole deleteReverseById(UUID id);
}

File: integration-tests-kotlin/src/main/kotlin/io/micronaut/jimmer/kotlin/it/service/IUserRoleService.kt
package io.micronaut.jimmer.kotlin.it.service

import io.micronaut.jimmer.kotlin.it.entity.UserRole
import java.util.UUID

interface IUserRoleService {
    fun findById(id: UUID): UserRole?

    fun updateById(id: UUID)

    fun deleteById(id: UUID)

    fun deleteReverseById(id: UUID): UserRole?
}
